puppet/modules/slave/manifests/docker.pp

```diff
@@ -17,4 +17,10 @@
       default  => 'docker',
     };
   }
+
+  group { 'docker':
+    ensure => 'present',
+  }
+
+  User<|title == jenkins|>{groups +> ['docker']}
```

Comment:
Does this add the jenkins user to the docker group if it's already not in it?

Reply:
It should. You may need to quote `jenkins` here.